### PR TITLE
meta: add action to update Uppy packages

### PR DESCRIPTION
**Pull request by aduh95**

### CODE REVIEW

File: .github/workflows/update-deps.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+name: Tools and deps update
+on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: 5 0 * * 0
```
Comment (Murderlon) [inline .github/workflows/update-deps.yml:5]: Do we need a schedule?

File: .github/workflows/update-deps.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - .github/workflows/update-deps.yml
```
Comment (Murderlon) [inline .github/workflows/update-deps.yml:11]: Maybe I misunderstand but don't we want to do this only before a deploy on `main`?

File: .github/workflows/update-deps.yml (hunk 1 above)
Comment (aduh95) [reply .github/workflows/update-deps.yml:5]: It should avoid us forgetting to run it manually.

File: .github/workflows/update-deps.yml (hunk 2 above)
Comment (aduh95) [reply .github/workflows/update-deps.yml:11]: Hum I think we only want it after each Uppy release.

Comment (Murderlon) [reply .github/workflows/update-deps.yml:11]: But now it runs on every PR?

Comment (mifi) [reply .github/workflows/update-deps.yml:11]: maybe if we could have uppy publish to npm (uppy@latest) on every commit, then we could use that here so we could update uppy.io to uppy@latest using the workflow_dispatch whenever we need that (or the schedule will)

Comment (aduh95) [reply .github/workflows/update-deps.yml:11]: > But now it runs on every PR?

It runs on every PRs that modify `.github/workflows/update-deps.yml`.

**Review by Murderlon — APPROVED** [review]

**Pull request merged**